sdk/core/azure-core/inc/azure/core/modified_conditions.hpp

```diff
@@ -19,7 +17,7 @@ namespace Azure { namespace Core {
   /**
    * @brief Specifies HTTP options for conditional requests based on modification time.
    */
-  struct RequestConditions : MatchConditions
+  struct ModifiedConditions
```

Comment:
What motivated this change, given some of the other languages like .NET do it with the inheritance and call it `RequestConditions`?

https://github.com/Azure/azure-sdk-for-net/blob/b493f7ad5487fd0d01f004e9f7ebdf50bfec0b96/sdk/core/Azure.Core/src/RequestConditions.cs#L11

Reply:
Jeffrey said yes. https://github.com/Azure/azure-sdk-for-cpp/issues/1608

Reply:
Thanks for linking to the issue.